docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2196,6 +2196,8 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/audit/serviceProperties/mapper`|string|The AuditEventLog mapper that determines the output format - valid types are 'cadf' and 'auditevent'. 'auditevent' refers to the FHIR Resource AuditEvent, and 'cadf' refers to the Cloud logging standard.|
 |`fhirServer/audit/serviceProperties/load`|string|The location that the configuration is loaded from 'environment' or 'config'.|
 |`fhirServer/audit/serviceProperties/kafka`|object|A set of name value pairs used as part of the 'config' for publishing to the kafka service. These should only be Kafka properties.|
+|`fhirServer/audit/hostname`|string|A string used to identify the Hostname, useful in containerized environments|
```

Comment:
In our Cloud deploy, we use a different approach.  We have a configDropin that defines a 
`<variable name="defaultHostName">` and then we set this via environment variable.
Not sure if its better/worse, but just something to consider.

Reply:
I added code to pick that off if its set

Reply:
marking as resolved